tests/py/dynamo/partitioning/test_dynamic_partitioning.py

```diff
@@ -53,51 +50,6 @@ def forward(self, x):
             1,
             f"Number of TRT segments should be 1 but got {trt_segments}",
         )
```

Comment:
Test case with assume_dynamic_shape_support=True flag was removed. TestDynamicPartitioning test checks if there is only trt segment in module.